### PR TITLE
Add multiple ingresses usage section

### PR DESCRIPTION
**Pull request by flmmartins**
Signed-off-by: Fernanda Martins <flmmartins@gmail.com>

### CODE REVIEW

**Comment by flmmartins** [comment]
Originated from issues:
#https://github.com/jetstack/cert-manager/issues/841
 #https://github.com/jetstack/cert-manager/issues/2271

**Comment by flmmartins** [comment]
assign @munnerz

File: content/en/docs/usage/ingress.md [hunk 1]
```diff
@@ -85,6 +85,11 @@ trigger `Certificate` resources to be automatically created:
   returned.  This is useful for keeping compatibility with the `ingress-gce`
   component.
 
+## Generate multiple certificates with multiple ingresses
+
+If you need to generate certificates from multiple ingresses make sure it has the issuer annotation.
+Besides the annotation is necessary that each ingress posess a unique `tls.secretName`
```
Comment (maelvls) [inline content/en/docs/usage/ingress.md:91]:
```suggestion
If you need to generate certificates from multiple ingresses make sure it has the issuer annotation.
Besides the annotation, it is necessary that each ingress possess a unique `tls.secretName`.
```

**Review by maelvls — CHANGES_REQUESTED** [review]
Thanks for the contribution! Here is one tiny suggestion.

File: content/en/docs/usage/ingress.md (hunk 1 above)
Comment (inteon) [reply content/en/docs/usage/ingress.md]: I applied these suggestion & rebased this PR.

Comment (maelvls) [reply content/en/docs/usage/ingress.md]: Woah, 2021! 😅🤯

Screenshot 2023-09-05 at 10 55 47

**Comment by maelvls** [comment]
Thanks for updating the PR, Tim!

/lgtm

**Pull request merged**